Add GetFile documentation to Using App Services guides (#171)
smartdevicelink/sdl_app_library_guides: docs/Other SDL Features/Using App Services/index.md

});
sdlManager.sendRPC(performAppServiceInteraction);
```
!@

### 5. Getting a File from a Service Provider

> In some cases, a service may upload an image that can then be retrieved from the module. To get the image name you need to retrieve, you will need to have the `SDLAppServiceData` available (see point 2 above).

Reviewer: ```suggestion
In some cases, a service may upload an image that can then be retrieved from the module. First, you will need to get the image name from the @![iOS]`SDLAppServiceData`!@@![android,javaSE,javaEE]`AppServiceData`!@ (see point 2 above). Then you will use the image name to retrieve the image data. 
```

@![iOS]
##### Objective-C
```objc

> SDLAppServiceData *data = <#Get the App Service Data#>;

Reviewer: ```suggestion
SDLAppServiceData *data = <#Get the App Service Data#>;
```

> SDLWeatherData *weatherData = data.weatherServiceData; // Assuming this contains weather service data

Reviewer: ```suggestion
SDLWeatherServiceData *weatherData = data.weatherServiceData;
```

    SDLGetFileResponse *getFileResponse = response;
    if (getFileResponse == nil || !response.success) {
        // Something went wrong, examine the resultCode and info
        return;
    }

    NSData *rpcImageData = response.bulkData;
    if (imageData == nil) {
        // There's no image data
        return;
    }

    [imageData appendData:rpcImageData];
    imageDataLengthReceived += rpcImageData.length;
    if (getFileResponse.offset == 0 && getFileResponse.length != nil) {
        imageDataLength = getFileResponse.length.unsignedIntegerValue;
    }

    if (imageDataLengthReceived < imageDataLength) {
        // Send additional GetFile requests to get the rest of the data using the offset parameter
    } else {
        // The file is complete, turn the file data into an image and use it.
    }
}];
```

##### Swift
```swift
let data: SDLAppServiceData = <#Get the App Service Data#>
let weatherData: SDLWeatherServiceData = data.weatherServiceData
guard let currentForecastImage = weatherData.currentForecast?.weatherIcon else {

>         // The image doesn't exist, exit early

Reviewer: ```suggestion
    // The image doesn't exist, exit early
```

>         return

Reviewer: ```suggestion
    return
```

}
let currentForecastImageName = currentForecastImage.value
let getCurrentForecastImage = SDLGetFile(fileName: currentForecastImageName)

> let imageDataLength = 0

Reviewer: ```suggestion
var imageDataLength = 0
```

> let imageDataLengthReceived = 0

Reviewer: ```suggestion
var imageDataLengthReceived = 0
```
